src/controllers/cartController.ts

```diff
@@ -0,0 +1,55 @@
+import { Request, Response } from 'express';
+import * as cartService from '../services/cartService';
+import { getUserId, sendErrorResponse, sendSuccessResponse } from '../helpers/http';
+import Joi from 'joi';
+import { cartSchema } from '../schemas/cart.schema';
+import { ERRORS_MESSAGES } from '../constants/errors';
+
+export const getCart = async (req: Request, res: Response) => {
+  const userId = getUserId(req);
+
+  try {
+    const cart = await cartService.getCart(userId);
+
+    sendSuccessResponse(res, cart);
+  } catch (error) {
+    sendErrorResponse(res, error as Error);
+  }
+};
+
+export const updateCart = async (req: Request, res: Response) => {
+  const userId = getUserId(req);
+
+  try {
+    Joi.assert(req.body, cartSchema, new Error(ERRORS_MESSAGES.PRODUCTS_INVALID));
```

Comment:
api is different to PUT /cart Swagger specification, could be added as separate route